examples/spring-boot-on-knative-example/readme.md

```diff
@@ -0,0 +1,59 @@
+# Spring Boot on Knative 
+
+The purpose of this example is to demonstrate the following:
+
+- How you can use the knative-spring-stater.
+- How dekorate detects that this is a web app and automatically configures services and probe.
+- How you can end-to-end test the application.
+- How you can trigger a docker build after the compilation.
+
+
+The application is using:
+
+    <dependency>
+      <groupId>io.dekorate</groupId>
+      <artifactId>knative-spring-starter</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+Which contains all the required modules, including the annotation processors that detect spring web applications.
+
+The [Main.class](src/main/java/io/dekorate/example/Main.java) is annotated with `@KnativeApplication` which triggers the resource generation.
+It's also annotated with `@EnableDockerBuild`. This annotation allows the user to trigger a docker build after the compilation, by passing the system property 
```

Comment:
I thought this, the `@EnableDockerBuild` is not needed anymore, is it?

Reply:
Good catch!